src/pipecat/utils/text/base_text_aggregator.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class BaseTextAggregator(ABC):
+    @abstractmethod
+    def aggregate(self, text: str) -> Optional[str]:
+        pass
+
+    @abstractmethod
+    def handle_interruption(self):
+        pass
+
+    @abstractmethod
+    def reset(self) -> str:
```

Comment:
Just wondering about the name, should it be something like `get_and_reset` or `retrieve_and_reset` to make it clear that this is how we retrieve the aggregated text ?

Reply:
I ended up creating a property `text` and `reset()` doesn't return anything.